go.mod

```diff
@@ -43,11 +41,81 @@ require (
 	golang.org/x/sync v0.0.0-20220513210516-0976fa681c29
 	golang.org/x/term v0.0.0-20210927222741-03fcf44c2211
 	golang.org/x/time v0.0.0-20210723032227-1f47c861a9ac
-	golang.org/x/tools v0.1.10 // indirect
 	google.golang.org/grpc v1.47.0
 	gopkg.in/yaml.v3 v3.0.1
 )
 
+require (
```

Comment:
I was curious as to why we had a second `require` section with so many indirect dependencies. For reference, the answer is here https://go.dev/ref/mod:

> At go 1.17 or higher:
> - The go.mod file includes an explicit [require directive](https://go.dev/ref/mod#go-mod-file-require) for each module that provides any package transitively imported by a package or test in the main module. (At go 1.16 and lower, an [indirect dependency](https://go.dev/ref/mod#glos-direct-dependency) is included only if [minimal version selection](https://go.dev/ref/mod#minimal-version-selection) would otherwise select a different version.) This extra information enables [module graph pruning](https://go.dev/ref/mod#graph-pruning) and [lazy module loading](https://go.dev/ref/mod#lazy-loading).
> - Because there may be many more // indirect dependencies than in previous go versions, indirect dependencies are recorded in a separate block within the go.mod file.
> - go mod vendor omits go.mod and go.sum files for vendored dependencies. (That allows invocations of the go command within subdirectories of vendor to identify the correct main module.)
> - go mod vendor records the go version from each dependency’s go.mod file in vendor/modules.txt.